packages/react-storage/src/components/StorageBrowser/actions/handlers/listLocationItems.ts

```diff
@@ -1,10 +1,19 @@
+import { ListPaginateWithPathOutput } from 'aws-amplify/storage';
```

Comment:
We only be using the internals types since they are aligned with the internals functions

Reply:
fixed

Reply:
🍕🍕🍕🍕🍕🍕

Reply:
⭐ suck

Reply:
Trust erodes by disrespecting people and asking them to eat pizza everynight

Reply:
@reesscot

Reply:
The best man